app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  before_save{ self.email = email.downcase }
```

Comment:
No. Convention: Do not add white spaces before and after [] () {} means the code like following 
```
before_save {self.email = email.downcase}
```